sup3r/preprocessing/data_handling.py

```diff
@@ -561,10 +561,8 @@ def __init__(self, file_paths, features, target=None, shape=None,
             bad_shape = (sample_shape[0] > self.grid_shape[0]
                          and sample_shape[1] > self.grid_shape[1])
             if bad_shape:
-                msg = (f'spatial_sample_shape {sample_shape[:2]} is larger '
-                       f'than the raster size {self.grid_shape}')
-                logger.warning(msg)
-                warnings.warn(msg)
+                logger.debug(f'spatial_sample_shape {sample_shape[:2]} is '
```

Comment:
@bnb32  i reduced the severity of these warnings, they're really just in the wrong place and generate a lot of noise. Thoughts?

Reply:
Yeah totally agree with this